string command
float64 pose
---
bool success
